vignettes/tutorial2.Rmd

```diff
@@ -0,0 +1,447 @@
+---
+title: "Fldgen v2 Tutorial - Joint Temperature and Precipitation emulation and function how to"
+author: "Abigail Snyder"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Fldgen Tutorial}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  fig.width=8,
+  cache=FALSE,
+  comment = "#>"
+)
+library('fldgen')
+library('dplyr')
+library('ggplot2')
+library('reshape2')
+dofieldplots <- require('gcammaptools')
+```
+
+The fldgen package allows you to ingest temperature and precipitation output from an earth system model (ESM) and generate randomized temperature fields that have the same space and time correlation properties as the original ESM data.  This tutorial focuses on how to use the functions in the package to generate and analyze joint temperature and precipitation fields.  The details of how the method works are covered in a companion paper.  
+
+All of the functions used here are documented in R's help system.  Since our purpose here is to outline what functions have to be called, and in what sequence, to perform the analysis, we haven't repeated material from the help files.  If you're confused about how a function is supposed to work, consult the help files.  For example, `help(read.temperatures)` will print the docs for the function that reads the netCDF temperature fields.
+
+## Using the field generator
+
+### Setup
+
+```{r params}
```

Comment:
Since you repeat this block and the next further down, you could probably delete them here.  You don't use them in the one-step tutorial anyhow.